.github/workflows/eslint-workflow.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Lint checking for typescript'
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    name: 'Checking Lint'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Install dependencies
+        run: pnpm install
```

Comment:
I commented about this in https://github.com/suspensive/react/pull/19#discussion_r1103780362, please check it out.